### ISSUE TITLE
java.lang.ClassCastException at minechem.utils.Recipe.init(Recipe.java:142) ~[Recipe.class:?] 

### ISSUE THREAD

**Issue by s1mples** [jakimfett/Minechem#813]
MC 1.7.10
forge-1.7.10-10.13.4.1448-1.7.10.jar

3 mods:
BetterBuildersWands-0.4.1-1.7.10r62+87fcd65.jar
CoFHLib-[1.7.10]1.0.3-175.jar
Minechem-1.7.10-5.0.5.406.jar

Crash at start_new/load world.

Client full log at http://rghost.ru/6BQwnstSs.

**Comment by ThexXTURBOXx** [comment]
Still exists :/ I will try to create a Pull Request, if I manage to fix this.

**Comment by Blue64** [comment]
> File is deleted

I was going to look into it for ideas, but I guess I can't.

**Comment by ThexXTURBOXx** [comment]
Well, yeah :D Just try it yourself, maybe it crashes for you, too

**Comment by Blue64** [comment]
I don't have Better Builders Wands Mod lol I do however ironically have the other two

**Comment by ThexXTURBOXx** [comment]
And for you it works? :D

**Comment by ThexXTURBOXx** [comment]
https://github.com/iopleke/Minechem/blob/master/src/main/java/minechem/utils/Recipe.java
When we look there in line 142, there is a cast from an array to a list (I think so, don't remember the methods fully right now). We should be better using Arrays.asList() then :D

**Comment by Blue64** [comment]
yeah, it works for me, the bug is between MineChem and Better Builders Wands.

**Comment by Blue64** [comment]
I believe they [changed it](https://github.com/iopleke/Minechem/commit/e7aff2d0a7338e9266f031b5ad3c4d7685679361?diff=unified) from something similar specifically for [Line 142](https://github.com/iopleke/Minechem/blob/master/src/main/java/minechem/utils/Recipe.java#L142) for [that section](https://github.com/iopleke/Minechem/blob/master/src/main/java/minechem/utils/Recipe.java#L140-L142) of [that chunk](https://github.com/iopleke/Minechem/blob/master/src/main/java/minechem/utils/Recipe.java#L115-L146)

**Comment by ThexXTURBOXx** [comment]
Hmm... Yeah, looks like they did, but they didn't release the new version on curseforge. :(

**Comment by Blue64** [comment]
well, perhaps when they do a major bug-fix (such as my "`potion.`" crash during `PostInit`), they'll do a Push for a release `:D` (Fingers crossed)

**Comment by ThexXTURBOXx** [comment]
Would be awesome :D But it pretty looks like, that they discontinued the mod :(

**Comment by Blue64** [comment]
got a response from the old Project Lead @jakimfett stating that he plans to still provide Jenkins Builds once he gets his Server up and running, so if we can fix the Bug in the Source Code, he might release it as an Update.

**Comment by jakimfett** [comment]
The tl;dr version is:  
I moved to a new place, so my server rack has been in varying stages of disarray for the past bit.

Everything should (in theory) be back up and running now, and if someone makes a PR I should be able to have it built and released as an alpha update within a day or two, maybe a week tops if stuff is *really* broken behind the scenes.
